sci-electronics/gspeakers/gspeakers-0.11-r3.ebuild

```diff
@@ -1,18 +1,16 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit autotools flag-o-matic gnome2
 
 DESCRIPTION="GTK based loudspeaker enclosure and crossovernetwork designer"
-HOMEPAGE="http://gspeakers.sourceforge.net/"
-SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
-
+HOMEPAGE="https://gspeakers.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/gspeakers/${PN}/${P}/${P}.tar.gz"
```

Comment:
This was fine the way it was. There usually is an empty line between S/SRC_URI and LICENSE.